playbooks/ansible/roles/sit.xfs/tasks/new_volume.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: Unmount the filesystem if already mounted
+  mount:
+    path: "{{ path }}"
+    state: unmounted
+
```

Comment:
I think it would be better to put this in the caller (playbooks/ansible/roles/samba.setup/tasks/main.yml). This way it will be useful for all backends (though some might need additional steps, but those could be added in future patches)

Reply:
Xavi, I did some tests with the umount before mkfs. 
I did the following
1) Run "make" which brings up the cluster
2) when the tests are running on the client, I interrupt and
3) rerun make

At this point, I suspect samba still is actively operating on the test filesystem causing the unmount operation to fail. If we wait a while and re-run, it works successfully and completes. 

To get around this, we need to stop samba before we attempt the unmount. Since this is handled by ctdb, it complicates matters.

I think the easier option for now to allow us to re-run make on the cluster is to simply use the option to not force the mkfs for xfs. I have pushed the original patches one more time. Let me know what you think.